docs/qwik-testing-library/example.mdx

```diff
@@ -0,0 +1,101 @@
+---
+id: example
+title: Example
+sidebar_label: Example
+---
+
+Below are some examples of how to use the Qwik Testing Library to tests your
+Qwik components.
+
+You can also learn more about the [**queries**][tl-queries-docs] and [**user
+events**][tl-user-events-docs] to help you write your tests.
+
+[tl-queries-docs]: ../queries/about.mdx
+[tl-user-events-docs]: ../user-event/intro.mdx
+
+## Qwikstart
+
+This is a minimal setup to get you started, with line-by-line explanations.
+
+```tsx title="counter.spec.tsx"
+// import qwik-testing methods
+import {screen, render, waitFor} from '@noma.to/qwik-testing-library'
+// import the userEvent methods to interact with the DOM
+import {userEvent} from '@testing-library/user-event'
+// import the component to be tested
+import {Counter} from './counter'
+
+// describe the test suite
+describe('<Counter />', () => {
+  // describe the test case
+  it('should increment the counter', async () => {
+    // render the component into the DOM
+    await render(<Counter />)
+
+    // retrieve the 'increment count' button
+    const incrementBtn = screen.getByRole('button', {name: /increment count/})
+    // click the button twice
+    await userEvent.click(incrementBtn)
+    await userEvent.click(incrementBtn)
+
+    // assert that the counter is now 2
+    await waitFor(() => expect(screen.getByText(/count:2/)).toBeInTheDocument())
```

Comment:
We prefer using `findByText` in this case.
The result will be the same, but the intention is clearer.

Reply:
wow I wasn't aware `findBy` queries actually had a retry strategy! good to know, I'll update it :) thanks!